src/Mvc/test/WebSites/BasicWebSite/Program.cs

```diff
@@ -1,24 +1,34 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.IO;
+using System.Threading.Tasks;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
 
 namespace BasicWebSite
 {
     public class Program
     {
-        public static void Main(string[] args) => CreateWebHostBuilder(args).Build().Run();
+        public static Task Main(string[] args)
+        {
+            var host = CreateHostBuilder(args)
+                .Build();
 
-        // Do not change. This is the pattern our test infrastructure uses to initialize a IWebHostBuilder from
-        // a users app.
```

Comment:
Is this comment still valid?

Reply:
We still support both patterns (WebHostBuilder hasn't gone away) so we should keep coverage for this.

Reply:
So even though it's obsolete we still need to make sure it keeps working.